src/browser/FileProxy.js

```diff
@@ -56,11 +57,6 @@
     var File = require('./File');
 
     (function (exports, global) {
-        var indexedDB = global.indexedDB || global.mozIndexedDB;
```

Comment:
Removed this check for Firefox( OS?). Caniuse is pretty green in terms of IndexedDB: https://caniuse.com/#feat=indexeddb

Reply:
If desired, I'll re-add the IndexedDB check and adjust the error message.

Reply:
Do not remove it in this PR.